CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to abn
+
+This outlines how to propose a change to abn.
+
+## The "Pull request" process
+
+In general all edits to the abn package go through the process of a
+[Pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+
+### Fixing typos
+
+If you spot typos or other language related errors in the documentation you may
+directly use the GitHub web interface to edit the source file.
+Simply click on the edit :pen: button and Github will guide you through the
+process of forking the repository and creating a pull request.
+
+### Beyond typos
```

Comment:
I would add headers to the paragraphs down here.

Reply:
https://github.com/furrer-lab/abn/pull/82/commits/8f55fac1881dc1352b1439d11a7116c9ade55989